iot_verifier/src/tx_scaler.rs

```diff
@@ -75,17 +76,14 @@ impl Server {
         let refresh_start = Utc::now();
         tracing::info!("density_scaler: generating hex scaling map, starting at {refresh_start:?}");
         let mut global_map = GlobalHexMap::new();
-        let active_gateways = self
+        let gateways = self
```

Comment:
this could still stay `active_gateways` since it's the list of gateways that beaconed within the verifier's db within the inactivity threshold window